extensions/wikia/ThemeDesigner/ThemeSettings.class.php

```diff
@@ -13,6 +13,7 @@ class ThemeSettings {
 
 	const WordmarkImageName = 'Wiki-wordmark.png';
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this constant "WordmarkImageName" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS115)